### PR TITLE
Adding CSP to meta tag in head.html

### LINKED ISSUE #223: Move CSP from Header to Meta tags
Move CSP to the <meta> tag under the <head> element in the HTML

Advantages:
- Simple code change to update the CSP instead of having to deal with tickets to update CDN configs.
- Header size will not be large and cause problems in the future

Disadvantages:
- Content-Security-Policy delivery through HTTP response supports some extra features compared to delivery via a HTML meta element, such as Content-Security-Policy-Report-Only and report-uri, frame-ancestors, and sandbox directives.

Since we aren't using any of the above that are not supported in the HTML meta element, we should be good to do this.

### PR DESCRIPTION
**Pull request by amol-anand**
## Issue
Fix #223 

## Test URLs
  
- Before: https://main--merative2--hlxsites.hlx.page/
- After: https://csp-test--merative2--hlxsites.hlx.page/
  
## Description
Moving CSP from response header to show up in the <meta> element in the HTML directly

### CODE REVIEW

**Review by helms-charity — CHANGES_REQUESTED** [review]
I'm seeing a different Referrer Policy now, is this a problem?
image

File: head.html [hunk 1]
```diff
@@ -1,5 +1,6 @@
...o.com sjrtp7.marketo.com cdn.cookielaw.org resources.digital-cloud-west.medallia.com *.kampyle.com js.driftt.com; img-src 'self' data: *.merative.com *.mktoweb.com *.google.co.in *.google.com *.google.ca *.googletagmanager.com *.112.2o7.net *.ytimg.com assets.adobedtm.com dpm.demdex.net cdn.cookielaw.org cm.everesttech.net *.medallia.com *.kampyle.com; frame-src 'self' *.mktoweb.com *.navattic.com *.adsrvr.org www.youtube.com player.vimeo.com *.merative.com documentservices.adobe.com merative.demdex.net *.medallia.com *.kampyle.com; connect-src 'self' *.mktoweb.com *.mktoresp.com *.marketo.com *.salesloft.com js.zi-scripts.com *.zoominfo.com *.adsrvr.org *.112.2o7.net assets.adobedtm.com viewlicense.adobe.io *.onetrust.com merative.tt.omtrdc.net analytics.google.com stats.g.doubleclick.net dpm.demdex.net cdn.cookielaw.org *.medallia.com *.kampyle.com; style-src 'self' 'unsafe-inline' *.mktoweb.com *.marketo.com go.merative.com assets.adobedtm.com cdn.cookielaw.org *.medallia.com *.kampyle.com; font-src 'self' *.mktoweb.com assets.adobedtm.com cdn.cookielaw.org *.medallia.com *.kampyle.com; object-src 'none'; manifest-src 'self'; media-src 'self' *.merative.com; worker-src 'none';">
```
Comment (helms-charity) [inline head.html:1]: Do you mind changing the order of the 1st two entries (or something) just so we can confirm which CSP is taking precedence?

Comment (amol-anand) [reply head.html:1]: You can have multiple CSPs and the URL has to pass through all the CSPs for it to work.
https://w3c.github.io/webappsec-csp/#multiple-policies

**Comment by amol-anand** [comment]
> I'm seeing a different Referrer Policy now, is this a problem? image

Don't see that in my testing. the headers look exactly the same between main and csp-test branches.

**Comment by helms-charity** [comment]
I have not seen that difference again after viewing different pages/different branches, so I think it was a false alarm.

**Review by helms-charity — APPROVED** [review]

**Pull request merged**